docs/operation/operation.md

```diff
@@ -932,6 +932,7 @@ while choosing the endpoint for the given request
 A set of metrics will be exposed to track passive health check:
 
 * `passive-health-check.endpoints.dropped`: Number of all endpoints dropped before load balancing a request, so after N requests and M endpoints are being dropped this counter would be N*M.
+* `passive-health-check.requests.passed`: Number of unique requests where PHC was able to avoid sending them to unhealthy endpoints.
```

Comment:
minor: you can link it to numbers mentioned in previous metrics and tell it's 'N' actually.

Reply:
I think adding the variable here will make a confusion.